.github/workflows/build.yml

```diff
@@ -63,3 +69,45 @@ jobs:
 
       - name: Run check
         run: if ! meson test -C build; then cat build/meson-logs/testlog.txt; false; fi
+
+  build-windows:
+    runs-on: windows-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { sys: mingw32 }
```

Comment:
mingw32 is deprecated upstream.

Reply:
That may be, but people may still be using it, so it's still useful to test.